contentcuration/contentcuration/static/js/edit_channel/exercise_creation/hbtemplates/assessment_item_answer_list.handlebars

```diff
@@ -7,6 +7,7 @@
 		<div class="col-sm-11">ANSWERS</div>
 	{{/if}}
 </div>
+{{#if input_answer}}<div class="input_question_prompt">NOTE: Equivalent formats are treated as the same value (1 1/2 = 3/2 = 1.5)</div>{{/if}}
```

Comment:
Small banner to indicate formats will be evaluated based on value. Does this wording make sense or is there a better way of phrasing this?